recipes/corona_lineage_dynamics/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+CONDA_PREFIX=$PREFIX
+g++ --std=c++17 scripts/count_lineages_country.cpp -o scripts/count_lineages_country -O2 -lstdc++fs -lboost_regex -I ${CONDA_PREFIX}/include/ -L ${CONDA_PREFIX}/lib/
+
+Rscript -e 'install.packages("devtools", repos="https://cloud.r-project.org");
+    '
+
+Rscript -e 'install.packages(c("XML", "binom", "plotly", "htmlwidgets", "countrycode", "doParallel",
```

Comment:
Please use `requirements > run` to add the dependencies

Reply:
I moved several of them into the requirements, but it seems we cannot move all of them.